test/fixtures/esnext/es6-classes/getter-setter-super.js

```diff
@@ -12,6 +20,18 @@ class Cat extends Animal {
   get sound() {
     return super.sound + ' MEOW!';
   }
+
+  get name() {
+    return super.name;
+  }
```

Comment:
Should this be necessary? Without a name getter declared, `cat.name` always returns `undefined`. Everything works fine if a name property is defined in the immediate parent.